frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -31,8 +31,8 @@ export function getMetabaseSdkCssVariables(theme: MantineTheme, font: string) {
     :root {
       --mb-default-font-family: ${font};
       ${getSdkDesignSystemCssVariables(theme)}
-      ${getThemeSpecificCssVariables(theme)}
       ${getDynamicCssVariables(theme)}
+      ${getThemeSpecificCssVariables(theme)}
```

Comment:
This allows `theme.components.question.toolbar.backgroundColor` to override the auto-derived colors.